pandas/compat/__init__.py

```diff
@@ -162,6 +164,10 @@ def bytes_to_str(b, encoding='ascii'):
     def signature(f):
         return inspect.getargspec(f)
 
+    # See also: https://bugs.python.org/issue34145
```

Comment:
what is the issue?

Reply:
Python2 uuid3 is incompatible with unicode input. Python3 uuid3 is incompatible with non-unicode input. The link in the comment is clear and also gives the same work around I currently have implemented.

Reply:
ok by the time we merge this we won't support PY2 any longer, so pls indicate that this can then be removed.

Reply:
Isn't that already clear from the fact that it is in the compat module? It is surrounded by code that is only there to support PY2. In fact that is stated 10 lines above this function declaration. (I know you probably don't see this in the code review.)